enterprise_access/apps/subsidy_request/tasks.py

```diff
@@ -144,3 +144,93 @@ def send_admins_email_with_new_requests_task(enterprise_customer_uuid):
 
     customer_config.last_remind_date = datetime.now()
     customer_config.save()
+
+
+@shared_task(base=LoggedTaskWithRetry)
+def send_learner_credit_admins_email_with_new_requests_task(policy_uuid, enterprise_customer_uuid):
+    """
+    Task to send new learner credit request emails to admins.
+
+    This task can be manually triggered from browse_and_request when a new
+    LearnerCreditRequest is created. It will send the latest 6 requests in
+    REQUESTED state to enterprise admins.
+
+    Args:
+        policy_uuid (str): subsidy access policy uuid identifier
+        enterprise_customer_uuid (str): enterprise customer uuid identifier
+    Raises:
+        HTTPError if Braze client call fails with an HTTPError
+    """
+
+    # Get only the latest 6 LearnerCreditRequest objects in REQUESTED state for this customer
+    subsidy_model = apps.get_model('subsidy_request.LearnerCreditRequest')
+    subsidy_requests = subsidy_model.objects.filter(
+        enterprise_customer_uuid=enterprise_customer_uuid,
+        state=SubsidyRequestStates.REQUESTED,
+    ).order_by("-created")[:6]
```

Comment:
Since we are redirecting the admin on a specific budget page, we should filter the requests here on `policy_uuid`. This will prevent confusion if a customer has multiple policies configured with B&R.